scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,267 @@ output:
 
-For today's example, we are particularly interested in comparing the gene expression across cell types in samples from patients diagnosed with ARMS vs. samples from patients diagnosed with ERMS.
-To help us out later on when we are performing our analysis and making plots, let's incorporate the information in this sample metadata table into the `colData` of the integrated dataset.
+```{r print reducedDim names}
+# look at the names of the dimensionality reduction present in the SCE object
+reducedDimNames(integrated_sce)
+```
 
-```{r}
-# add the sample metadata to the coldata 
+When we look at the assay names we should see that there are 3 matrices, `counts`, `logcounts`, and `fastmnn_corrected`. 
+The `counts` and `logcounts` assays correspond to the uncorrected gene expression data that has been merged but NOT integrated.
+The `fastmnn_corrected` data contains the corrected gene expression data obtained from integration. 
+For this exercise we will not be using the `fastmnn_corrected` data (more on why not once we get to setting up the differential expression), but we need to be aware that it is present and be able to distinguish it from our uncorrected data. 
+
+In the `reducedDim` slots you should see `PCA` and `UMAP` which both correspond to the pre-integrated dimensionality reduction results.
+You should also see `fastmnn_PCA` and `fastmnn_UMAP` reduced dimensions, which correspond to the integrated results.
+
+Just like in the integration notebook, this dataset also contains the cell type annotations found in the `celltype_fine` and `celltype_broad` columns of the `colData`.
+These cell types were originally assigned in [Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003).
+We will use these cell type assignments to set up the DE analysis below, but they are not required.
+It's important to note that DE analysis can be applied to any subpopulation of interest that is shared across samples and does not need to be explicitly defined by a cell type.
+
+Because we are going to be doing DE analysis between ARMS and ERMS samples, let's start by labeling cells in the integrated dataset based on their RMS subtype.
+To do this we will need to be sure that the subtype is present in the `colData` of the integrated SCE object.
+If it's not there, we need to add it in.
+
+```{r coldata head}
+# look at the head of the coldata
+head(colData(integrated_sce))
+```
+
+Fortunately we also have the sample metadata table that we read in earlier, which contains information about each of the samples present in the dataset.
+
+
+```{r sample metadata}
+# print out sample metadata
+head(sample_metadata)
+```
+
+Looking at this sample table, we see a column named `subdiagnosis` which accounts for the RMS subtype, ARMS or ERMS.
+We also see other columns that contain information about each specific sample and may contribute to differences seen across the samples.
```

Comment:
"and may contribute to differences seen across the samples" seems like overkill here, or we would want to actually explore it?